0800-appendix-8530-sandstone.Rmd

```diff
@@ -147,7 +153,7 @@ moderate rearing potential.
 
 Electrofishing was conducted with results summarised in Tables
 \@ref(tab:tab-fish-site-8530) - \@ref(tab:tab-fish-dens-8530) and Figure
-\@ref(fig:plot-fish-box-8530). Dolly varden and cutthroat trout were caught both downstream and upstream of the culvert, while rainbow trout were only caught upstream. 
+\@ref(fig:plot-fish-box-8530). Dolly varden and cutthroat trout were caught both downstream and upstream of the culvert, while rainbow trout were only caught upstream `r if(gitbook_on){knitr::asis_output("(Figures \\@ref(fig:photo-8530-07) - \\@ref(fig:photo-8530-08)).")}else(knitr::asis_output("(Figure \\@ref(fig:photo-8530-u01))."))` 
```

Comment:
because we have info about upstream vs downstream in the name of the photo itself I would not include it in the chunk name.  It makes each memo a bit less abstract and a bit trickier to use as a template for the next site down the line